jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -295,6 +298,11 @@ public void invoke(final long window, final boolean focus) {
         showWindow();
 
         allowSwapBuffers = settings.isSwapBuffers();
+
+        // Create OpenCL
+        if (settings.isOpenCLSupport()) {
+            initOpenCL(window);
+        }
```

Comment:
Any specific reason why this should now only happen when restarting and not as part of the general init process?  Wouldn't this prevent openCL init when regularly starting?

Reply:
It is part of the general init process. This is called in createContext which in turn is used by the restart and the initial context.